tests/instruments/hp/test_hp8116a.py

```diff
@@ -23,244 +23,39 @@
 #
 
 import pytest
-import math
-import time
-from pymeasure.instruments.hp import HP8116A, hp8116a
 
-pytest.skip('Only work with connected hardware', allow_module_level=True)
+from pymeasure.test import expected_protocol
 
+from pymeasure.instruments.hp import HP8116A as _HP8116A
```

Comment:
I'm curious: why this? Can't you do something like
```
HP8116A.status = property(fget=lambda: Status(5))
```

Reply:
That's more elegant. Thanks for that suggestion